Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3-slim-buster
+FROM python:3.7-slim-buster
```

Comment:
👍 on this. Just had an issue caused by the `python:3-*` images moving up to 3.8 recently.

Reply:
Yap, same here.